build-config/src/main/resources/weld-checkstyle/checkstyle.xml

```diff
@@ -50,9 +50,9 @@
             <property name="ignoreNumbers" value="-1, 0, 1, 2, 3"/>
         </module>
         <module name="MissingSwitchDefault" />
-        <module name="RedundantThrows">
+        <!--<module name="RedundantThrows">
```

Comment:
Yea I checked the code with this rule allowed...
It would be a terrific ordeal to correct all those checkstyle errors. We agreed that it is simply **way too strict** and we won't go for it.